distribution/changelog.txt

```diff
@@ -2,6 +2,7 @@
 ------------------------------------------------------------------------
 - Fix: [#15096] Crash when placing entrances in the scenario editor near the map corner.
 - Improved: [#3417] Crash dumps are now placed in their own folder.
+- Fix: [#14316] Closing the Track Designs Manager window causes broken state.
```

Comment:
We group the changelog by type, then by issue number. Hence, your entry should be at the top.